docs/_documentations/troubleshooting.md

```diff
@@ -390,6 +390,33 @@ These steps reproduce the issue:
 
 **Workaround** Run the `Attach Debugger` action manually.
 
+<!--
+Codewind version: 0.6.0
+Action/Topic: Appsody with Codewind
+Issue type: bug/info
+Issue link: https://github.com/eclipse/codewind/issues/1061
+-->
+## Appsody binds fail
+
+If an Appsody respository is already added in your local Appsody CLI, and if you try to add the Appsody repository again with a different name or ID, the Appsody bind might fail in the following scenario:
+1. Use a template manager to add the Appsody repository in Codewind.
```

Comment:
Not "a" template manager, it's the template manager (or whatever we call it officially) that's in VSCode and Eclipse

Reply:
@makandre I will change "a" to "the." But when will we have the official name for the template manager? Or do you know who I could ask to get this info?

Reply:
Maybe check with @jopit on that

Reply:
I'd say something like `Use 'Manage Template Sources' in the IDE to...`